concordia/static/scss/base.scss

```diff
@@ -811,9 +811,10 @@ body.view-transcriptions--asset-detail main {
 
 .help-center-card {
     color: #fff;
-    width: 300px;
+    // width: 300px;
     height: 225px;
     background-color: var(--primary-color);
+    margin-bottom:1rem;
```

Comment:
Follow the instructions for enabling pre-commit: https://github.com/LibraryOfCongress/concordia/blob/master/docs/how-we-work.md#configure-your-local-checkout-with-code-quality-hooks

This will make sure that every commit runs Prettier even if made using an editor which doesn't have Prettier enabled.

Reply:
ok got pre-commit installed now.